lib/paper_trail/model_config.rb

```diff
@@ -150,17 +142,17 @@ def setup_versions_association(klass)
       klass.has_many(
         klass.versions_association_name,
         lambda do
-          relation = order(model.timestamp_sort_order)
-          item_type = klass.paper_trail.send(:versions_association_item_type)
-          relation = relation.unscope(where: :item_type).where(item_type: item_type)
-          relation
+          order!(model.timestamp_sort_order)
```

Comment:
`order!` is tagged `:nodoc:` meaning that it is private API. The rails team could remove it at any time without warning. I don't think we should use it.